pymorphy3/utils.py

```diff
@@ -134,7 +130,7 @@ def kwargs_repr(kwargs=None, dont_show_value=None):
     kwargs = kwargs or {}
     dont_show_value = set(dont_show_value or [])
     return ", ".join(
-        "%s=%s" % (k, repr(v) if k not in dont_show_value else "<...>")
+        f'{k}={repr(v) if k not in dont_show_value else "<...>"}'
```

Comment:
I'd not recommend to put such big expressions into f-string.

Reply:
Using format() instead of f-string only here would be inconsistent. To solve the problem of the long expression, simply introducing another intermediate variable would do, which is also more readable I suppose.

Reply:
Ok, intermediate variable is ok